system/services/dataExport/DataExportService.cfc

```diff
@@ -699,6 +715,31 @@ component {
 		return arguments.objectName & "__" & arguments.fieldName;
 	}
 
+	private struct function _processRelatedFieldExportDetail(
+		  required struct  propAttributes
+		,          boolean objectAllowExpandFields = false
+	) {
+		var processed = {
+			  shouldExpand = false
+			, expandFields = []
+		};
+
+		if ( StructKeyExists( arguments.propAttributes, "dataExportExpandFields" ) ) {
+
+			if ( IsBoolean( arguments.propAttributes.dataExportExpandFields ) ) {
+				processed.shouldExpand = arguments.propAttributes.dataExportExpandFields;
+			} else if ( ListLen( arguments.propAttributes.dataExportExpandFields ) > 0 ) {
```

Comment:
I think this simplifies to:

`} else if ( Len( arguments.propAttributes.dataExportExpandFields ) ) {`